usaspending_api/common/tests/test_drf_tracking.py

```diff
@@ -7,6 +7,7 @@
 import json
 
 
+@pytest.mark.skip(reason="Table logging is now disabled")
```

Comment:
We may want DB logging in the future, hence why I opted to leave it in and just skip it. I can remove it if we really want to.